api/paidAction/zap.js

```diff
@@ -78,10 +95,14 @@ export async function onPaid ({ invoice, actIds }, { models, tx }) {
       SELECT COALESCE(SUM(msats), 0) as msats
       FROM forwardees
     ), recipients AS (
-      SELECT "userId", msats FROM forwardees
+      SELECT "userId", msats, msats AS "stackedMsats" FROM forwardees
```

Comment:
`recipients."stackedMsats"` is never used

Reply:
That's a bug!